ee/tabby-webserver/email_templates/test.html

```diff
@@ -0,0 +1,2 @@
+Test
+This is a test email
```

Comment:
```suggestion
Your Tabby mail server is ready to Go!
---
Hello,

This is a test email from Tabby to confirm that your mail server configuration is correct. If you have received this email, it means that your configuration was successful. Thank you for using Tabby!

Best regards,
The Tabby Team
```

Let's use `---` as delimiter to separate title and body